client/reader/sidebar/expandable-heading.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import noop from 'lodash/utility/noop';
+
+/**
+ * Internal dependencies
+ */
+import SidebarHeading from 'layout/sidebar/heading';
+import Gridicon from 'components/gridicon';
+import Count from 'components/count';
+
+const ExpandableSidebarHeading = ( { title, count, onClick } ) => (
+	<SidebarHeading onClick={ onClick }>
```

Comment:
Thanks @mtias. I'm definitely in favour of extracting this to `layout/sidebar`, but probably a separate PR would be best at this point.